### ISSUE TITLE
Part of a Platform? (y/n)

### ISSUE THREAD

**Issue by notadevinnz** [TeamTopologies/Team-API-template#4]
What does the section "Part of a Platform? (y/n)" mean? What's the question here? What would need to be true for a team to answer Yes or No to this question?

**Comment by dinkargupta** [comment]
If the team is part of the platform offering for the company and offers service/facilitation/collaboration possibilities for other to benefit. I have just setup a team like that. It's an enabling team within service delivery platform that is facilitating automation of GRC controls on the platform and in future may offer conpliance check services for platform users (stream aligned domain teams). This team's consumers are platform engineers, developers and risk mgmt function.

Hope that is helpful.

**Comment by notadevinnz** [comment]
Hi Dinkar. I appreciate the quick response, but it doesn't clarify the situation for me. Team Topologies talks about four team types - Stream, Enabling, Sub-system, and Platform. The simplest interpretation of the question could be 'Is the the team a Platform team?' However, if that's the intent, why isn't the question "Which type of team is it"? 
Also, why is it phrased "Part of a Platform" rather than "Is the team a Platform team"?

**Comment by dinkargupta** [comment]
fair point. IMHO, this could be to signify that there will be product centric service model on this team's APIs (platform as a product), especially because as per TT's description of platform team, "it is a grouping of other team types that provide a compelling internal product …"

**Comment by matthewskelton** [comment]
Hi @notadevinnz @dinkargupta thank you for this discussion. Yes, in the 2019 TT book we talk about a "Platform team" but really this is more like a grouping of other teams (almost always). With hindsight, we should have called this a "Platform group", not "Platform team". 😔

So the question about being part of a platform shows this thinking: is the team part of a "Platform group"? 

Hope this helps!

**Comment by dinkargupta** [comment]
@matthewskelton that is precisely how we are using it. it's a "Service delivery platform" for us. a portfolio team. under that there are internal stream aligned, enablement and potentially a CSS team.

**Comment by notadevinnz** [comment]
@matthewskelton Can you clarify the relationship between platform teams and a platform group.  Is every team within a platform group a platform team?

**Comment by matthewskelton** [comment]
Hi @notadevinnz great question. The way I think about this now is that there is no such thing as a "platform team" but instead there are teams that might be part of a platform group. We might find Stream-aligned teams, Enabling teams, Complicated Subsystem teams, and other platform groups inside a platform [group]. Does that help?

**Comment by notadevinnz** [comment]
So does that mean that the question "Part of a Platform? (y/n)" should be read to mean "Part of a Platform **Group**? (y/n)"?

**Comment by matthewskelton** [comment]
@notadevinnz yes, that's how I would word it now, I think. Sounds like a pull request 🤓
